src/migrations/20230119000000-create-groups.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable max-len */
+module.exports = {
+  up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(async (transaction) => {
+    const loggedUser = '0';
+    const sessionSig = __filename;
+    const auditDescriptor = 'RUN MIGRATIONS';
+    await queryInterface.sequelize.query(
+      `SELECT
+          set_config('audit.loggedUser', '${loggedUser}', TRUE) as "loggedUser",
+          set_config('audit.transactionId', NULL, TRUE) as "transactionId",
+          set_config('audit.sessionSig', '${sessionSig}', TRUE) as "sessionSig",
+          set_config('audit.auditDescriptor', '${auditDescriptor}', TRUE) as "auditDescriptor";`,
+      { transaction },
+    );
+
+    await queryInterface.createTable('Groups', {
+      id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+      name: {
+        type: Sequelize.TEXT,
+        unique: true,
```

Comment:
if 2 users name a group the same thing, how are we imagining that will work?

Reply:
good question

Reply:
Since we don't have generic collaborators yet, and phase 1 of groups are private, I moved the userId to the groups table for the time being. Will adjust later to do user ownership better once we have generic collaborators.

Reply:
Does it make sense to make names unique via an index? (id, name)